src/libs/tools/src/plugindatabase.cpp

```diff
@@ -367,5 +382,12 @@ std::string MockPluginDatabase::lookupInfo (PluginSpec const & spec, std::string
 
 	return "";
 }
+
+PluginDatabase::func_t MockPluginDatabase::getSymbol (PluginSpec const & spec ELEKTRA_UNUSED,
+						      std::string const & which ELEKTRA_UNUSED) const
+{
+	// TODO implement mockup
+	return NULL;
+}
```

Comment:
At the moment there is no use for the mocked method. @markus2330 any advice for a good implementation or should I just leave it as it is?